py2pack/templates/opensuse.spec

```diff
@@ -122,9 +122,9 @@ CHOOSE: %pytest OR %pyunittest -v OR CUSTOM
 %python_alternative %{_bindir}/{{ script }}
 {%- endfor %}
 {%- if has_ext_modules %}
-%{python_sitearch}/*
```

Comment:
This is certainly a step in the right direction. I would prefer even
```rpm
%{python_sitearch}/{{name}}
%{python_sitearch}/{{name}}-%{version}*-info
```
We usually don’t need anything more than these two, so there is no need for the star in the end, and including the version number makes this implicitly check for correctly generated metadata (it is 0.0.0 otherwise).